low-level/files/opt/seagate/sspl/bin/sspl_setup_init.py

```diff
@@ -145,30 +168,28 @@ def process(self):
         # Create /tmp/dcs/hpi if required. Not needed for '<product>' role
         if self.role != "cortx":
             try:
-                os.makedirs('/tmp/dcs/hpi', mode=0o777, exist_ok=True)
+                os.makedirs(self.hpi_path, mode=0o777, exist_ok=True)
                 zabbix_uid = self.get_uid('zabbix')
                 if zabbix_uid != -1:
-                    os.chown('/tmp/dcs/hpi', zabbix_uid, -1)
+                    os.chown(self.hpi_path, zabbix_uid, -1)
             except OSError as error:
-                print(error) 
+                print(error)
 
         # Check for sspl required processes and misc dependencies like installation, etc based on 'role'
-        if self.role : 
+        if self.role:
             self.check_dependencies(self.role)
         
         # Create mdadm.conf to set ACL on it.
-        with open('/etc/mdadm.conf', 'a'):
-            os.utime('/etc/mdadm.conf')
+        with open(self.mdadm_path, 'a'):
+            os.utime(self.mdadm_path)
 
-        # self._send_command('setfacl -m u:sspl-ll:rw /etc/mdadm.conf') 
-        #  What should be the file permissions need here and are `setfacl` and below impl equivalent??
-        os.chmod('/etc/mdadm.conf', mode=0o666)
+        # self._send_command('setfacl -m u:sspl-ll:rw /etc/mdadm.conf')
+        os.chmod(self.mdadm_path, mode=0o666)
```

Comment:
sure on chmod replacement ? I was asking to use same setfacl through subprocess

Reply:
Not sure, but setfacl command is not working using subprocess it gives 'Operation not supported' error, so I need to search for the solution, didn't got it straight forward.

Reply:
keep a TODO note here to find replacement for setfacl
And use chmod to give rw permission for sspl-ll user